velox/exec/OutputBuffer.cpp

```diff
@@ -556,7 +592,10 @@ bool OutputBuffer::deleteResults(int destination) {
     }
     freed = buffer->deleteResults();
     dataAvailable = buffer->getAndClearNotify();
+    VELOX_CHECK_LT(destination, finishedBufferStats_.size());
+    finishedBufferStats_[destination] = buffers_[destination]->stats();
     buffers_[destination] = nullptr;
```

Comment:
nit: is it better to provide a finish method for destination buffer stats to call finish() before set it to finishedBufferStats_? Thanks!

Reply:
done.